docs/docs/dcap.md

```diff
@@ -1,78 +1,80 @@
+
+This is the recommended way to use Conclave on Azure virtual machines, but may also be used to host enclaves outside
+the Azure cloud. It requires no additional setup.
+
+If using the Azure DCAP Client and PCCS, you may also want to set the Azure DCAP client logging level to FATAL, as the 
+default setting is quite verbose:
 
 ```sh
 export AZDCAP_DEBUG_LOG_LEVEL=FATAL
 ```
 
-### Intel DCAP Client
+Other configuration options for the Azure DCAP client are documented [here](https://github.com/microsoft/Azure-DCAP-Client).
 
-Intel provides a DCAP client as part of the DCAP runtime. You can use this option if you want to run your code on 
-bare-metal machines and don't want to rely on cloud providers.
+### Intel DCAP Client & PCCS
 
-To use Intel's DCAP plugin, you need to install the DCAP client package. For example, in Ubuntu, you can 
-use the following command:
+Intel provides reference DCAP client and PCCS implementations. If you do not want to rely on externally hosted services
+for attestation, such as the Azure PCCS, you can set up your own PCCS based on the Intel implementation. This requires 
+that you create an account with Intel and generate an API key so that your PCCS can communicate with the Intel 
+provisioning servers.
 
-`sudo apt-get install libsgx-dcap-default-qpl`
+Full Installation instructions for the Intel PCCS and DCAP client can be found on 
+[Intel's website](https://www.intel.com/content/www/us/en/developer/articles/guide/intel-software-guard-extensions-data-center-attestation-primitives-quick-install-guide.html).
 
-You also need a [subscription](https://api.portal.trustedservices.intel.com/provisioning-certification) to use 
-Intel's DCAP plugin. You must also set up a Provisioning Certificate Caching Service (PCCS). Intel provides an 
-example and some instructions [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs/README.md). 
-If you like to use Intel's reference implementation of their [PCCS service](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs),
-then you might need to provide data in the correct format for Conclave. Please [let us know](mailto:conclave@r3.com) if 
-you need to set this up.
+Once installed, Conclave will load the Intel DCAP client rather than the bundled Azure client.
 
-### Azure DCAP Client
+This is the recommended approach for applications that will run outside the Azure cloud.
 
-Microsoft provides an Azure DCAP client, which you can use on Azure virtual machines. Azure's DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) 
-does not require any subscription to Intel services. 
+### Azure DCAP Client
 
-To use an Azure DCAP client, you need to install Microsoft's package list and Azure's DCAP client package. For example,
-in Ubuntu, you can use the following commands:
+Although Conclave bundles the Azure DCAP client, it may also be installed directly from Microsoft's repositories:
```

Comment:
This does raise the question of why have this section if the plugin is already provided?

Reply:
Could be useful if someone needs a newer version of the Azure DCAP plugin than we bundle.

Reply:
OK